view/frontend/templates/config.phtml

```diff
@@ -19,18 +19,22 @@
     <?php
     $frontName = $block->getRequest()->getFrontName();
     $pathInfo = $block->getRequest()->getPathInfo();
-    ?>
-<script>
+    $config = $block->getJsonConfig();
 
+    $script = <<<script
     require (['uiRegistry'], function(registry) {
-        registry.set('amazonPay', <?= /* @noEscape */ $block->getJsonConfig() ?>)
+        registry.set('amazonPay', {$config});
     });
+    script;
+    ?>
 
     <?php if ($frontName != 'checkout' || (strpos($pathInfo, 'checkout/cart') !== false)): ?>
-    require (['Amazon_Pay/js/model/storage'], function(amazonStorage) {
-        amazonStorage.clearAmazonCheckout();
-    });
+        <?php
+        $script .= PHP_EOL . "require (['Amazon_Pay/js/model/storage'], function(amazonStorage) {
```

Comment:
Instead of appending to the string here, would it be better to just have two echos for each of the requires, and secure render them?

Reply:
Possibly? I can poke at that and see if it seems cleaner